pkgs/development/zig-packages/build-zig-package/default.nix

```diff
@@ -0,0 +1,58 @@
+{ lib, stdenv, zig, mktemp }:
+
+{ strictDeps ? true
+, depsBuildBuild ? [ ]
+, nativeBuildInputs ? [ ]
+, configurePhase ? null
+, buildPhase ? null
+, checkPhase ? null
+, installPhase ? null
+, enableParallelBuilding ? true
+, meta ? { }
+, ...
+}@attrs:
+
+stdenv.mkDerivation (attrs // {
+  inherit strictDeps enableParallelBuilding;
+  depsBuildBuild = [ ] ++ depsBuildBuild;
+  nativeBuildInputs = [ zig ] ++ nativeBuildInputs;
+
+  configurePhase =
+    if (configurePhase == null) then ''
+      runHook preConfigure
+      runHook postConfigure
+    '' else
+      configurePhase;
+
+  postConfigure = ''
+    export XDG_CACHE_HOME=$(mktemp -d)
+  '';
+
+  buildPhase =
+    if (buildPhase == null) then ''
+      runHook preBuild
+      zig build --verbose
+      runHook postBuild
+    '' else
+      buildPhase;
+
+  checkPhase =
+    if (checkPhase == null) then ''
+      runHook preCheck
+      runHook postCheck
+    '' else
+      checkPhase;
+
+  installPhase =
+    if (installPhase == null) then ''
+      runHook preInstall
+      zig build --prefix $out install
+      runHook postInstall
+    '' else
+      installPhase;
+
+  meta = meta // {
+    platforms = meta.platforms or zig.meta.platforms;
+    maintainers = (meta.maintainers or [ ]) ++ [ lib.maintainers.iogamaster ];
```

Comment:
not sure if it makes sense to add yourself too all packages using buildZigPackage

Reply:
I thought this meant I am maintaining the build-zig-package file.

Reply:
I removed myself